bench/runner.py

```diff
@@ -2,14 +2,17 @@
 # Copyright 2009 National Research Foundation (South African Radio Astronomy Observatory)
```

Comment:
Please check all these new copyright headers that have been added - the year must match when the file was first created.  Looking at the [history](https://github.com/ska-sa/katcp-python/commits/2ddcc4589ed9a3060c8d6ce2910beebce44c8a68/bench/runner.py) of this file, it only appeared in 2010.